src/assets/data/edition/series/1/section/5/intro.json

```diff
@@ -934,13 +934,13 @@
+                        "<p class='no-indent spacebreak'>Webern completed his continuity draft on 8 October and dated it thus (<span class='link-todo'><strong>M 321 Sk3</strong> mm. 21B–26B</span>). He wrote to Jone on 17 October to report on its completion, and also on his having begun work on “Des Herzens Purpurvogel” M 322.<sup><a id='note-ref-168' (click)=\"ref.navigateToIntroFragment({complexId: '', fragmentId: 'note-168'})\" title='Webern to Jone, 17 October 1934 (A-Wn, HS 431/1–139): “‘Sterne, Ihr silbernen Bienen’ is finished. Now I am busy with [‘]Des Herzens Purpurvogel fliegt durch Nacht’[.]”'>168</a></sup> He dated the beginning of an initial continuity draft “24. X[.] 34”, though he later broke off work on it (<a (click)=\"ref.selectSvgSheet({complexId: 'op25', sheetId: 'M_322_Sk3a'})\"><strong>M 322 Sk3</strong> mm. &#123;1A&#125;, &#123;1B&#125;, &#123;1C-2A&#125;, 1D–2B</a>). He had presumably begun the preceding sketches (<a (click)=\"ref.selectSvgSheet({complexId: 'op25', sheetId: 'M_322_Sk1'})\"><strong>M 322 Sk1</strong></a> and <a (click)=\"ref.selectSvgSheet({complexId: 'op25', sheetId: 'M_322_Sk2'})\"><strong>M 322 Sk2</strong></a>) before writing to Jone on 17 October; he might also have begun <a (click)=\"ref.selectSvgSheet({complexId: 'op25', sheetId: 'M_322_Sk3a'})\"><strong>M 322 Sk3</strong> measures &#123;1A&#125;, &#123;1B&#125;, &#123;1C–2A&#125;</a> before that date. Webern seems to have been working on the final continuity draft in early November,<sup><a id='note-ref-169' (click)=\"ref.navigateToIntroFragment({complexId: '', fragmentId: 'note-169'})\" title='Webern to Jone, 9 November 1934 (A-Wn HS 431/1–140): “‘Des Herzens Purpurvogel’ will soon be finished. I was just held up a little recently.”'>169</a></sup> which was completed on 15 November according to the date in the sketchbook (<span class='link-todo'><strong>M 322 Sk4</strong> m. 21C</span>).</p>",
```

Comment:
`<a (click)=\"ref.selectSvgSheet({complexId: 'op25', sheetId: 'M_322_Sk3a'})\"><strong>M 322 Sk3</strong> measures &#123;1A&#125;, &#123;1B&#125;, &#123;1C–2A&#125;</a>`

In dt. Version habe ich die Taktangabe aus dem Link rausgenommen. Hier auch oder oben wieder einfügen?

Reply:
Gute Idee: habe die Taktangabe jetzt hier auch rausgenommen.